pennylane/devices/default_qubit.py

```diff
@@ -591,13 +591,14 @@ def _setup_execution_config(self, execution_config: ExecutionConfig) -> Executio
         """
         updated_values = {}
 
-        jax_interfaces = {qml.math.Interface.JAX, qml.math.Interface.JAX_JIT}
-        updated_values["convert_to_numpy"] = (
-            execution_config.interface not in jax_interfaces
-            or execution_config.gradient_method == "adjoint"
-            # need numpy to use caching, and need caching higher order derivatives
-            or execution_config.derivative_order > 1
-        )
+        # uncomment once compilation overhead with jitting improved
```

Comment:
Can you add a `TODO` with a link to a shortcut story?

Reply:
added